tests/cypress/tests/unit/table_update_test.ts

```diff
@@ -0,0 +1,27 @@
+import shouldComponentUpdate from 'dash-table/components/Table/shouldComponentUpdate';
+
+describe('shouldComponentUpdate', () => {
+    it('should update on undefined -> defined props', () => {
+        assert(shouldComponentUpdate({}, { a: 0 }, {}, {}));
+    });
+
+    it('should update on undefined -> defined state', () => {
+        assert(shouldComponentUpdate({}, {}, {}, { a: 0 }));
```

Comment:
This test fails without the added `...nextProps` https://github.com/plotly/dash-table/pull/785/commits/8849e85932ba6b4f3d739a3a6934fbb852d90115#diff-aa707fa377faf64638e062a12fb52d85R13